karma-test-shim.js

```diff
@@ -30,35 +30,31 @@ System.config({
 
 System.config(
 {
+  paths: {
+    // paths serve as alias
+    'npm:': 'node_modules/'
+  },
   map: {
-    'rxjs': 'node_modules/rxjs',
-    '@angular': 'node_modules/@angular',
-    firebase: 'node_modules/firebase/firebase.js',
-    'app': 'dist'
+    'app': 'dist',
+    '@angular/core': 'npm:@angular/core/bundles/core.umd.js',
```

Comment:
Can SystemJS load the ES modules instead, using the old configuration but adding `format: 'esm'`?